Code/LauncherUnified/launcher_generator.cmake

```diff
@@ -16,7 +16,7 @@ set_property(GLOBAL PROPERTY LAUNCHER_UNIFIED_BINARY_DIR ${CMAKE_CURRENT_BINARY_
 # When using an installed engine, this file will be included by the FindLauncherGenerator.cmake script
 get_property(O3DE_PROJECTS_NAME GLOBAL PROPERTY O3DE_PROJECTS_NAME)
 foreach(project_name project_path IN ZIP_LISTS O3DE_PROJECTS_NAME LY_PROJECTS)
-    
+
```

Comment:
intentional newline?

Reply:
No, there is was just trailing whitespace on the current line.
I trimmed the trialing whitespace from all lines of the file using the VSCode "Trim Trialing Whitespace" option